regex-parsing/src/test/resources/finders/SuperfluousCurlyBraceFinder.yml

```diff
@@ -0,0 +1,14 @@
+- '(abc){1}' # Noncompliant {{Remove this unnecessary quantifier.}}
+- '(abc){1,1}' # Noncompliant {{Remove this unnecessary quantifier.}}
```

Comment:
What about  `(abc){4,4}` I think this rule should also include these cases. WDYT?

Reply:
Discussed on slack. We agreed that this looks like a legit rule, but not like one named "SuperfluousCurlyBraceFinder" ...